docs/source/usage/api.rst

```diff
@@ -280,3 +280,19 @@ This is for the legacy, AoS + SoA particle containers only:
 .. autoclass:: amrex.space3d.Particle_2_1
    :members:
    :undoc-members:
+
+.. _usage-api-eb:
+
+Embedded Boundaries
+-------------------
+
+Embedded boundary (EB) support is still experimental. To build pyAMReX with
```

Comment:
Although we use EBs in WarpX very well with Python... Maybe one can skip the first sentence completely or rephrase to:
```suggestion
Embedded boundary (EB) support in pyAMReX is still minimal. To build pyAMReX with
```

We only need more EB bindings if we build and allocate EBs from pyAMReX. If we just interact with EB-enabled simulations, we are already quite good (e.g., `iMultiFab` et al. is what we need for WarpX with EB).

Reply:
Updated